content/v2.0/reference/api/influxdb-1x/dbrp.md

```diff
@@ -0,0 +1,155 @@
+menu:
+  v2_0_ref:
+    name: DBRP mapping
+    parent: 1.x compatibility
+weight: 302
+products: [cloud]
+related:
+  - /v2.0/reference/api/influxdb-1x/query
+  - /v2.0/reference/api/influxdb-1x/write
+  - /v2.0/api/#tag/DBRPs, InfluxDB 2.0 API /dbrps endpoint
+---
+
+The InfluxDB 1.x data model includes [databases](https://docs.influxdata.com/influxdb/v1.8/concepts/glossary/#database)
+and [retention policies](https://docs.influxdata.com/influxdb/v1.8/concepts/glossary/#retention-policy-rp).
+InfluxDB 2.0 replaces both with [buckets](/v2.0/reference/glossary/#bucket).
+To support InfluxDB 1.x query and write patterns in InfluxDB 2.0, databases and retention
+policies are mapped to buckets using the **database and retention policy (DBRP) mapping service**.
+
+The DBRP mapping service uses the **database** and **retention policy** specified in
+[compatibility API](/v2.0/reference/api/influxdb-1x/) requests to route operations to a bucket.
+[Create DBRP mappings automatically](#automatically-create-dbrp-mappings) with the `/write` compatibility endpoint or
+[create DBRP mappings manually](#manually-create-and-manage-dbrp-mappings) using the `/api/v2/dbrps` API endpoint.
+
+- [DBRP mapping behavior](#dbrp-mapping-behavior)
+- [Automatically create DBRP mappings](#automatically-create-dbrp-mappings)
+- [Manually create and manage DBRP mappings](#manually-create-and-manage-dbrp-mappings)
+
+### Default retention policies
+A database can have multiple retention policies with one set as default.
+If no retention policy is specified in a query or write request, InfluxDB uses
+the default retention policy for the specified database.
+
+## DBRP mapping behavior
+
+- [When writing data](#when-writing-data)
+- [When querying data](#when-querying-data)
+
+### When writing data
+When writing data to InfluxDB 2.0 using the [`/write` compatibility endpoint](/v2.0/reference/api/influxdb-1x/write/),
+the DBRP mapping service uses the specified database and retention policy to:
+
+1. Check for a DBRP mapping that matches the database and retention policy.
+  - If a DBRP mapping exists, write data to the mapped bucket.
+  - If no mapping exists, then:
+2. Check to see if a bucket matching the naming convention, `<database>/<retention-policy>`, exists.
+  - If a bucket with that name exists, add the DBRP mapping to the existing bucket
+     and write the data to the bucket.
+  - If no bucket with that name exists, then:
+3. Create a new bucket using the naming convention `<database>/<retention-policy>`,
+   add the DBRP mapping to the new bucket, and write the data to the bucket.
+
+##### If no retention policy is specified:
+1. Check for the specified database in the DBRP mapping service.
+  - If the database exists, get the default retention policy for the specified database, and then:
+  - If the database doesn't exists, add the database, set its default retention policy as `autogen`, and then:
+2. Continue the [normal DBRP mapping behavior](#when-writing-data).
```

Comment:
the DBRP mapping service checks for a bucket mapped to the database and retention policy:

- If a mapped bucket is found, data is written to the bucket.
- If an unmapped bucket with a name matching:
    - **database/retention policy** exists, a DBRP mapping is added to the bucket, and data is written to the bucket.
    - **database** exists (without a specified retention policy), the default database retention policy is used, a DBRP mapping is added to the bucket, and data is written to the bucket.
- If no matching bucket is found, a new **database/retention-policy** bucket is created with a DBRP mapping, and data is written to the bucket. If no retention policy is specified, `autogen` is used.